packages/fxa-content-server/app/tests/spec/views/settings/change_password.jsx

```diff
@@ -0,0 +1,161 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import $ from 'jquery';
+import { assert } from 'chai';
+import AuthErrors from 'lib/auth-errors';
+import Broker from 'models/auth_brokers/base';
+import helpers from '../../../lib/helpers';
+import Metrics from 'lib/metrics';
+import Notifier from 'lib/channels/notifier';
+import React from 'react';
+import Relier from 'models/reliers/relier';
+import sinon from 'sinon';
+import User from 'models/user';
+import {ChangePasswordHeader, ChangePasswordForm} from 'views/settings/change_password';
+import View from 'views/settings/change_password';
+import {
+  render,
+  cleanup,
+  fireEvent,
+  queryByText,
+  getByText,
+} from '@testing-library/react';
+
+const { isEventLogged } = helpers;
+
+const EMAIL = 'a@a.com';
+
+describe.only('views/settings/change_password', function() {
+  var account;
+  var broker;
+  var metrics;
+  var notifier;
+  var relier;
+  var user;
+  var view;
+
+  beforeEach(function () {
+    notifier = new Notifier();
+    metrics = new Metrics({ notifier });
+    relier = new Relier();
+
+    broker = new Broker({
+      relier: relier
+    });
+
+    user = new User({});
+
+    view = new View({
+      broker: broker,
+      metrics: metrics,
+      notifier,
+      relier: relier,
+      user: user
+    });
+  });
+
+  afterEach(() => {
+    cleanup();
+  });
+
+  describe('ChangePasswordHeader Component', () => {
+    function renderChangePassword(){
+      render(<ChangePasswordHeader />);
+    }
+    it('renders the password header correctly', () => {
+      renderChangePassword();
+      const title = document.querySelectorAll('.settings-unit-title');
+      const settingsButtons = document.querySelectorAll('.settings-button');
+      assert.lengthOf(title, 1);
+      assert.include(title[0].innerText, 'Password');
+      assert.lengthOf(settingsButtons, 1);
+      assert.include(settingsButtons[0].innerText, 'Change');
+    });
+  });
+
+  describe('Change Password Form Component', function () {
+    beforeEach(function () {
+      account = user.initAccount({
+        email: EMAIL,
+        sessionToken: 'abc123',
+        verified: true
+      });
+      sinon.stub(account, 'isSignedIn').callsFake(function () {
+        return Promise.resolve(true);
+      });
+
+      sinon.stub(view, 'getSignedInAccount').callsFake(function () {
+        return account;
+      });
+
+      return view.render()
```

Comment:
You might be able to get away with just rendering the react component and not the view here.